core/templates/pages/exploration-editor-page/translation-tab/translator-overview/translator-overview.component.html

```diff
@@ -61,7 +61,8 @@
     margin-top: 8px;
   }
   .oppia-translation-progress-bar {
-    margin: 7px 0 7px 15px;
+    background-color: rgb(193, 233, 243);
```

Comment:
Can you please explain your reasoning behind choosing this color?

Reply:
It was just a deeper blue shade of the the original one, so selected this color

Reply:
Ok, can you please change the format from rbg to hex (#c1e9f3)?

Reply:
yes made the change, please take a look